spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -430,13 +314,11 @@ private RabbitTemplate buildRabbitTemplate(RabbitProducerProperties properties)
 		}
 		rabbitTemplate.afterPropertiesSet();
 		return rabbitTemplate;
+
 	}
 
-	private String getStackTraceAsString(Throwable cause) {
-		StringWriter stringWriter = new StringWriter();
-		PrintWriter printWriter = new PrintWriter(stringWriter, true);
-		cause.printStackTrace(printWriter);
-		return stringWriter.getBuffer().toString();
+	RabbitMessageChannelErrorConfigurer getErrorConfigurer(){
+		return (RabbitMessageChannelErrorConfigurer) this.errorConfigurer;
```

Comment:
I think we should be more strict here and require definitely only `RabbitMessageChannelErrorConfigurer` injection - no any generic `BinderErrorConfigurer` support.
More over you are going to catch here `ClassCast` if you don't provide exactly `RabbitMessageChannelErrorConfigurer`.